### ISSUE TITLE
[MPM] Deprecation error

### ISSUE THREAD

**Issue by bodhinandach** [KratosMultiphysics/GiDInterface#551]
@jginternational I just checked that in MPM problem types, there are deprecation warnings referring to the recent modification in the model.cpp:

https://github.com/KratosMultiphysics/Kratos/blob/e1e32a10f5dd699f358a22a0447ca2badba33528/kratos/containers/model.cpp#L131-L134

Basically just to change the name from `"Parts_Parts_Auto1"` to `"Initial_MPM_Material.Parts_Parts_Auto1"` with the parent model part.

Just want to confirm shortly are you aware of this? And shall we modify anything local in the MPM? Particularly, since we have two initial model parts created in GiD.

Many thanks.

**Comment by jginternational** [comment]
Yes, but it would be easier if MPM had an example running in the interface (check Fluid, Solid or any app, in the examples folder)
